### ISSUE TITLE
Color Contrast Ratio issue for Link

### ISSUE THREAD

**Issue by prativen** [OfficeDev/office-ui-fabric-react#3682]
### Bug Report

- __Package version(s)__: 5.38.1 
- __Browser and OS versions__: all

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes)

Requested priority: (Normal) 

Products/sites affected: (VSTS)

#### Describe the issue:

Link component does not meet the high contrast ratio. Its just below the 4.5:1 requirement.
https://contrastchecker.com/ reports failure for #0078D7 foreground against #FFFFFF background for fonts below 18 pts.

**Comment by betrue-final-final** [comment]
It actually passes this checker. Note that we build against the AA standard, and not the AAA standard. The AAA standard really is for high contrast themes, and not default configurations.
![image](https://user-images.githubusercontent.com/20363037/34731510-64efc554-f517-11e7-8a2a-9d1dbece650b.png)

**Comment by nikhilexe** [comment]
Actually its failing for small font text. see this link. It seems the above link is only checking for 18pt.

https://webaim.org/resources/contrastchecker/
![image](https://user-images.githubusercontent.com/25940082/34826545-f7e64d24-f6fc-11e7-8c75-ffcf3ea88223.png)

Check here also http://www.wyngcontrastchecker.com/

**Comment by betrue-final-final** [comment]
This checker is contradicting itself as 4.5:1 passes AA.

**Comment by betrue-final-final** [comment]
The above checker did check for smaller font text, if you look closer at the screenshot.
![image](https://user-images.githubusercontent.com/20363037/34836019-710d995c-f6ab-11e7-96dc-e10e01bbaddc.png

The other checker also shows 4.5:1
![image](https://user-images.githubusercontent.com/20363037/34836114-b85301a8-f6ab-11e7-9265-638cecad236c.png)

**Comment by nikhilexe** [comment]
I guess it could be precision issue, in one of the tool its giving 4.49:1, if you just changed one shade the foreground color to #0078D6 then it passed the test both place.

**Comment by betrue-final-final** [comment]
We actually had a long conversation about this earlier, involving a number of accessibility experts. It's actually 4.495:1. Our contrast experts stated making change to make it 4.5:1 is imperceptible to the naked eye. This color is the brand color for Outlook and one we use extensively in many of our components. Changing this would have a huge impact on many teams for little to no benefit other than a reading on a tool. Here is the comparison of these two colors, and also directly next to each other.

![image](https://user-images.githubusercontent.com/20363037/34839530-bd9b4f52-f6b6-11e7-889c-fcf1edf3b1cb.png)

**Comment by nikhilexe** [comment]
Thanks @betrue-final-final, given that its a brand color for outlook and used extensively in many components then it doesn't make sense to change it which is not easy to perceive.
